src/App.js

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
-import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import React from "react";
+import "./App.css";
+import { Route } from "react-router-dom";
+import { Home, About, Contact, Navigation } from "./components";
 
 const App = () => (
   <div>
     <Navigation />
+    <switch>
```

Comment:
switch is supposed to be capitalised `Switch`. And imported from `react-router-dom`